### PR TITLE
German localization

### PR DESCRIPTION
**Pull request by rabauke**
Changes in rpm/harbour-books.spec were necessary to build the rpm successfully on my installation of Qt Creator 3.5.0 (SailfishOS).

### CODE REVIEW

File: rpm/harbour-books.spec [hunk 1]
```diff
@@ -53,7 +52,7 @@ desktop-file-install --delete-original \
 %{_datadir}/icons/hicolor/*/apps/%{name}.png
 
 %check
-make -C test test
+# make -C test test
```
Comment (monich) [inline rpm/harbour-books.spec]: Why does that fail in your environment?

File: rpm/harbour-books.spec [hunk 2]
```diff
@@ -33,7 +33,6 @@ FBReader-based e-book reader.
 %setup -q -n %{name}-%{version}
 
 %build
-%qtc_qmake5 harbour-books.pro
```
Comment (monich) [inline rpm/harbour-books.spec]: Without this line it doesn't build in OBS

File: rpm/harbour-books.spec (hunk 1 above)
Comment (monich) [inline rpm/harbour-books.spec]: I did reproduce the failure when I tried to build the rpm from QtCreator. However, I would still like to keep the check step in the OBS build.

**Comment by monich** [comment]
I think you need to take `harbour-books.spec` part out of this PR if there's no way to keep both OBS and QtCreator happy.

**Comment by rabauke** [comment]
PR includes only changes related to the new translation now.

File: app/app.pro [hunk 3]
```diff
@@ -84,9 +84,10 @@ icon256.files = icons/256x256/harbour-books.png
 icon256.path = $$TARGET_ICON_ROOT/256x256/apps
 INSTALLS += icon256
 
-CONFIG += sailfishapp_i18n sailfishapp_i18n_idbased
+CONFIG += sailfishapp sailfishapp_i18n sailfishapp_i18n_idbased
```
Comment (monich) [inline app/app.pro]: I don't think this change is necessary, `sailfishapp` is already added to `CONFIG` at line 2.

Comment (rabauke) [inline app/app.pro]: I am sorry, I have missed this configuration in line 2. This unnecessary change has been removed now.

File: app/translations/harbour-books-de.ts [hunk 4]
```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="sv">
```
Comment (mschaaf) [inline app/translations/harbour-books-de.ts]: language="de" instead of sv i guess

Comment (monich) [inline app/translations/harbour-books-de.ts]: Nice catch (even though it probably doesn't really matter)

Comment (rabauke) [inline app/translations/harbour-books-de.ts]: Damned copy&paste. Thanks for spotting this issue.

**Comment by monich** [comment]
Thanks!
